test/data/configClient.test.js

```diff
@@ -1,7 +1,11 @@
+jest.mock('../../server/data/dataAccess/db')
```

Comment:
this would usually be hoisted to the top level . To do that with jest-ts we would have to add babel which bit reluctant to do